src/components/Footer.jsx

```diff
@@ -1,7 +1,40 @@
 import React from "react";
+import {
+  FaInstagram,
+  FaEnvelope,
+  FaSquareFacebook,
+  FaLinkedin,
+} from "react-icons/fa6";
+import Image from "next/image";
+import safLogo from "../app/saf-logo.webp";
+import Link from "next/link";
 
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div class="py-3 px-3 flex items-center justify-between bg-saf-beige-200">
+      <div>
+        <Image alt="saf-logo" style={{ width: "12%" }} src={safLogo} />
```

Comment:
don't use css, use tailwind:
Css is anything with style tag: `style={{ width: "12%" }}`
Tailwind is: `w-1/3` for example